src/site/content/en/learn/css/backgrounds/index.md

```diff
@@ -0,0 +1,456 @@
+  }
+  ```
+{% endAside %}
+
+
+### Setting a background-image with CSS gradient functions
+
+Several gradient CSS functions exist to allow you to generate a background-image, when passed two or more colors.
```

Comment:
Add backticks when you use `background-image` with a dash like this. This even works in the heading.